lib/services/bootstrap/pluginBootstrap.js

```diff
@@ -57,11 +57,11 @@ class PluginBootstrap extends SafeBootstrap {
       this.pluginName);
   }
 
-
+  // @todo use internalStorage.createCollection instead
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)